packages/components/src/index.ts

```diff
@@ -72,6 +74,7 @@ export * from './components/typography/Paragraph/Paragraph';
 export * from './components/typography/Text/Text';
 export * from './components/typography/TruncateWithTooltip/TruncateWithTooltip';
 export * from './components/Warning/Warning';
+export * from './components/Table/Table';
```

Comment:
btw we prefer to be specific here and not export everything

Reply:
Okey but why there is `export * from` almost everywhere 😄?

Reply:
Historic reasons :) but we will update it in the future.